IBF-Typhoon-model/src/typhoonmodel/pipeline.py

```diff
@@ -73,22 +77,40 @@ def main(path,remote_directory,typhoonname,LonMin,LatMin,LonMax,LatMax):
     TEST_REMOTE_DIR = '20210421120000'
     remote_dir = remote_directory
-    if not Activetyphoon:
-      logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")
-      if remote_dir is None:
-         remote_dir = TEST_REMOTE_DIR
-         #remote_dir='20210421120000' #for downloading test data otherwise set it to None
-      Activetyphoon=[typhoonname]  #name of typhoon for test
-      #logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")
-    elif remote_directory is not None:
-      logging.info(f"There is an active typhoon, but the user has requested to run for typhoon{typhoonname}")
-      #remote_dir='20210421120000' #for downloading test data otherwise set it to None
-      Activetyphoon=[typhoonname]  #name of typhoon for test
-      logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")    
+    if not Activetyphoon and debug_flag:
+        logging.info(f"No active typhoon in PAR, DEBUGGING piepline  for typhoon{typhoonname}")
+        Activetyphoon=[typhoonname]
+        if remote_dir is None:
+            remote_dir = TEST_REMOTE_DIR
+    elif not Activetyphoon and not debug_flag:
+        logging.info("No active typhoon in PAR stop pipeline")
+        print("currently no active typhoon in PAR DEBUG flag was set to False")
+        print("For Debugging you can set debug_flag=True via options")
+        exit()
     else:
-      logging.info(f"Running on active Typhoon(s) {Activetyphoon}")
-      #remote_dir=None # for downloading test data      Activetyphoon=['SURIGAE']
-    print("currently active typhoon list= %s"%Activetyphoon)
+        logging.info(f"Running on active Typhoon(s) {Activetyphoon}")
+        remote_dir=None # for downloading test data      Activetyphoon=['SURIGAE']
+        print("currently active typhoon list= %s"%Activetyphoon)
+        
+                
+ 
+
+    # if not Activetyphoon:
+    #   logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")      
+    #   if remote_dir is None:
+    #      remote_dir = TEST_REMOTE_DIR
+    #      #remote_dir='20210421120000' #for downloading test data otherwise set it to None
+    #   Activetyphoon=[typhoonname]  #name of typhoon for test
+    #   #logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")
+    # elif remote_directory is not None:
+    #   logging.info(f"There is an active typhoon, but the user has requested to run for typhoon{typhoonname}")
+    #   #remote_dir='20210421120000' #for downloading test data otherwise set it to None
+    #   Activetyphoon=[typhoonname]  #name of typhoon for test
+    #   logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")    
+    # else:
+    #   logging.info(f"Running on active Typhoon(s) {Activetyphoon}")
+    #   #remote_dir=None # for downloading test data      Activetyphoon=['SURIGAE']
+    # print("currently active typhoon list= %s"%Activetyphoon)
```

Comment:
You can delete this code that isn't used (it will be preserved in older commits on github)